src/search/landmarks/landmark_factory_rpg_sasp.cc

```diff
@@ -165,7 +165,7 @@ int LandmarkFactoryRpgSasp::min_cost_for_landmark(
       with "astar(lmcount(lm_rhw()))" on mystery/prob07.pddl in debug mode.
       See issue 467
     */
-    assert(min_cost < numeric_limits<int>::max());
+    //assert(min_cost < numeric_limits<int>::max());
```

Comment:
This is commented out because on the test instance for this issue this assertion fires (this is due to the unrelated issue467). It would be ideal if we merge issue467 before merging this, but we should at the very least make sure that both issues are merged during this sprint.

Reply:
I just commented this back in, because as you've written, this is not part of another issue. It will not influence the experiments as the experiments run in release mode anyway. I don't think it's necessary to wait for issue467 to be fixed first, and I don't think we should wait with merging once we agree this is ready to be merged.

Reply:
Good point. I'd still try to make sure issuei467 is merged soon after, but since this should easily be doable during this sprint I don't see a problem :)